src/Microsoft.AspNetCore.Mvc.Core/AuthorizeFilter.cs

```diff
@@ -10,7 +10,7 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Internal;
 
-namespace Microsoft.AspNetCore.Mvc.Internal
+namespace Microsoft.AspNetCore.Mvc
```

Comment:
`.Filters` is for filter interfaces/contracts/etc. Putting implementations there is a case of grouping by component type where we do grouping by feature in 99.9% of other cases.

> Is there a place where most other filters live?

Most live in `.Internal` because the attribute class can be used as the global filter.

The cors filters can go to `.Cors`

The best solution for authorization really seems to be carving out an `.Authorization` namespace and putting a few more things from internal in there.

@Eilon thoughts?